.travis.yml

```diff
@@ -9,13 +9,11 @@ notifications:
   email:
     on_failure: always
     recipients:
-      - 130s@2000.jukuin.keio.ac.jp
+      - smohaime@sfu.ca
```

Comment:
-1. You don't need to remove recipient addresses of someone else unless you're 100% sure.